app/styles/atoms/smart/logo.less

```diff
@@ -0,0 +1,236 @@
+:root {
+  --logo-color-stone-light: #f5f5f4;
+  --logo-color-stone-dark: #78716c;
+
+  --logo-color-rose-light: #ffe4e6;
+  --logo-color-rose-dark: #f43f5e;
+
+  --logo-color-orange-light: #ffedd5;
+  --logo-color-orange-dark: #f97316;
+
+  --logo-color-yellow-light: #fef3c7;
+  --logo-color-yellow-dark: #f59e0b;
+
+  --logo-color-lime-light: #ecfccb;
+  --logo-color-lime-dark: #84cc16;
+
+  --logo-color-emerald-light: #d1fae5;
+  --logo-color-emerald-dark: #10b981;
+
+  --logo-color-teal-light: #ccfbf1;
+  --logo-color-teal-dark: #14b8a6;
+
+  --logo-color-cyan-light: #cffafe;
+  --logo-color-cyan-dark: #06b6d4;
+
+  --logo-color-sky-light: #e0f2fe;
+  --logo-color-sky-dark: #0ea5e9;
+
+  --logo-color-blue-light: #dbeafe;
+  --logo-color-blue-dark: #3b82f6;
+
+  --logo-color-indigo-light: #e0e7ff;
+  --logo-color-indigo-dark: #6366f1;
+
+  --logo-color-violet-light: #ede9fe;
+  --logo-color-violet-dark: #8b5cf6;
+
+  --logo-color-purple-light: #f3e8ff;
+  --logo-color-purple-dark: #a855f7;
+
+  --logo-color-fuschia-light: #fae8ff;
+  --logo-color-fuschia-dark: #d946ef;
+
+  --logo-color-pink-light: #fce7f3;
+  --logo-color-pink-dark: #ec4899;
+
+  --logo-color-slate-light: #f1f5f9;
+  --logo-color-slate-dark: #64748b;
+}
```

Comment:
Are we sure we want to add these as variables ? 🤔 
I don't think we'll use these, as they are defined, elsewhere.